src/components/EntityInfo.vue

```diff
@@ -96,14 +96,21 @@
 
               <!-- Download Summary -->
               <span v-if="showDownloadSummaryBtn">
```

Comment:
Do the conditions in the getter need to be updated?

Do you need/want FF `download-summary-enabled` to be set for testing?

Any unit tests?

Reply:
Conditions are still good. 
We could* turn the flag on in DEV for UX to look at it, although it does nothing at this point.
Sounds like its coming up soon tho!

There is tests around its display currently. Nothing has changed there.

Reply:
OK, no changes then.

I know you can just email a screenshot to UX for their approval.